src/examples/mfem_common_plane.cpp

```diff
@@ -28,6 +28,17 @@
 
 #include <set>
 
+// Tribol includes
+#include "tribol/common/Parameters.hpp"
+#include "tribol/config.hpp"
```

Comment:
```suggestion
#include "tribol/config.hpp"
#include "tribol/common/Parameters.hpp"
```
It is generally a good idea to put the config header before all of yours in case it alters behavior.